daq_utils.py

```diff
@@ -124,6 +124,28 @@ def wave2energy(w, digits=2):
   else:
     return float(f"%.{digits}f" % (EV_ANGSTROM_CONSTANT/w))
 
+def gonio2lab(x_gonio, y_gonio, z_gonio, omega_deg):
```

Comment:
does this follow one of NSLS-II or beamline conventions for x, y, z?

Reply:
Unsure what the NSLS-II conventions are, I think it uses beamline conventions. X is the direction of the sample holder, Y is up and down and Z should be in the direction of the beam (sorry for the poor description)

Reply:
now that I think about it, I think this should be added to the general documentation, not what you are working on.

resolving